api/chat_test.go

```diff
@@ -19,6 +19,16 @@ func ChatRouterWrapper(r *gin.Engine) {
 	configGinChatRouter(r.Group("/api/chat"), dao.DaoWrapper{})
 }
 
+func ChatDefaultRouter(t *testing.T) func(r *gin.Engine) {
```

Comment:
```suggestion
func chatDefaultRouter(t *testing.T) func(r *gin.Engine) {
```
I guess this can also be not exported?